packages/@react-aria/dnd/stories/dnd.stories.tsx

```diff
@@ -113,9 +117,34 @@ storiesOf('Drag and Drop', module)
   .add(
     'Droppable grid',
     () => (
-      <Flex direction="row" gap="size-200" alignItems="center" wrap>
-        <DraggableCollectionExample />
-        <DroppableGridExample onDropEnter={action('onDropEnter')} onDropExit={action('onDropExit')} onDrop={action('onDrop')} />
+      <Flex direction="column" alignItems="start">
+        <ActionGroup
+          onAction={action => {
+            switch (action) {
+              case 'copy':
+              case 'cut': {
+                let selected = document.querySelector('[aria-label="Draggable list"] [aria-selected="true"] [role="gridcell"]') as HTMLElement;
+                selected?.focus();
+                document.execCommand(action);
+                break;
+              }
+              case 'paste': {
+                // This only works in Safari...
+                let selected = document.querySelector('[aria-label="List"] [aria-selected="true"] [role="gridcell"]') as HTMLElement;
+                selected?.focus();
+                document.execCommand('paste');
+                break;
+              }
+            }
+          }}>
+          <Item key="copy" aria-label="Copy"><Copy /></Item>
+          <Item key="cut" aria-label="Cut"><Cut /></Item>
+          <Item key="paste" aria-label="Paste"><Paste /></Item>
+        </ActionGroup>
+        <Flex direction="row" gap="size-200" alignItems="center" wrap>
+          <DraggableCollectionExample />
```

Comment:
Not super important and maybe something that can be fixed with the story, but I sometimes notice a focus ring flicker when tabbing from the copy/cut/paste buttons to the draggable collection items. It looks as if the drag handle of one of the items briefly gets a focus ring before the focus ring goes around the correct item

Reply:
yeah that's unrelated to this pr and has always been there a bit with collection components. I think there's a requestAnimationFrame somewhere that does the focusing and I guess that's a bit too long to wait...